### ISSUE TITLE
Sentence descriptive for each category on report card (tbc by Natasha)

### ISSUE THREAD

**Issue by LucieGueuning** [MSFREACH/msf-reach#267]
ACCESS: every information which is related to a physical access leading to a specific area or precise location such as road, airport or other. e.g.: plane not allowed to landed in airport, street over flood, groups blocked road, etc.
NEEDS: every information which is related to a need for a specific person or a wider range of population. e.g. shelters, water, medical care, etc. 
CONTACT: every information which is a contact to share with MSF staff. This contact can be helpful by being nearby the incident and providing more precise information or can be in needs. 
SECURITY: every information which is related to any type of security incident. Need to be flag as soon as possible to MSF staff.

**Comment by matthewberryman** [comment]
If the text is concise enough I could fit this onto that slide of the report card, else, we could add a help popup. Lucie let me know when the text is confirmed by Natasha.

**Comment by LucieGueuning** [comment]
Still under review for content

Need to be under a little 'i' info clickable (phone and desktop version) as below:
https://www.iconfinder.com/icons/339913/help_info_information_notice_icon#size=128

Devika confirmed for design?

**Comment by LucieGueuning** [comment]
@anexact

**Comment by anexact** [comment]
ACCESS 
Location / Damage

NEEDS
Population / Medical

CONTACT 
Details / Relevance 

SECURITY
Incident / Caution

**Comment by LucieGueuning** [comment]
FYI, message from Natasha below (to be resolved): 
Sun 11/03/2018 11:03
@anexact 

The categories look fine as described.  However, after having used the platform, I find 2 things missing
•	Response of other actors (e.g. a contact from Save the Children tell us that they are providing shelter in x location or a community member telling us the government is distributing NFIs in their village)
•	General info (e.g.  flood has receded in a location, volcano is currently erupting)
Should we just  have an “others” tab?  But the 4 tabs looks nice.  I’m not sure “looks nice” is one of the considerations in designing the platform though, ha ha.  Anyway, I know you and Etienne will think of something.

**Comment by anexact** [comment]
I think the modified proposal I have made covers both cases, but we can discuss again once Natasha reviews, yes?

etienne mobile
+49 176 830 46360

**Comment by LucieGueuning** [comment]
I don't think so. But you are right, let discuss it with her directly. I let you reply to her email to suggest your solution. Thanks. 

@anexact

**Comment by matthewberryman** [comment]
@LucieGueuning @anexact any update on this?

**Comment by LucieGueuning** [comment]
@anexact who took the lead on that to meet the need and follow-up the Natasha's comment

**Comment by anexact** [comment]
I remain opposed from UX/UI perspective to adding sentences to the buttons on the micro-survey. If they need sentences to describe them, the buttons have failed their purpose.

**Comment by matthewberryman** [comment]
I agree adding sentences is not good. But are we happy with 4 types?

**Comment by LucieGueuning** [comment]
I agree with sentences as well. 

4 types is ok. I got 2 months ago a feedback from users saying that sometimes they have info that is not fitting to a specific category. Can we add 'INFORMATION Any other type'

**Comment by anexact** [comment]
No, I strongly disagree. 
We must encourage users to use the category they think is most relevant, not constantly leave other options and loose threads. 
The 4 categories are more than sufficient once the platform is properly socialized in the organization.

**Comment by LucieGueuning** [comment]
SOLUTION: keep the categories as it is. the box (clickable) contact should lead to the contact card. 

No additional sentence neither a category named 'Information'. 

all clear?
